lib/process-services-cloud/src/lib/process/start-process/components/start-process-cloud.component.ts

```diff
@@ -130,6 +130,11 @@ export class StartProcessCloudComponent implements OnChanges, OnInit, OnDestroy
     isFormCloudLoading = false;
     processDefinitionLoaded = false;
 
+    displayStartProcessButton = true;
+    displayCancelButton = true;
```

Comment:
`display` is a synonym of `show`, you already have `showCancelButton` input property

Reply:
renamed to `showStartProcessButton`